projects/components/src/stepper/stepper.component.ts

```diff
@@ -48,29 +48,38 @@ import { StepperTabComponent } from './stepper-tab.component';
           </mat-step>
         </ng-container>
       </mat-stepper>
-      <div class="action-buttons">
-        <ht-button
-          class="cancel"
-          label="Cancel"
-          (click)="this.cancelled.emit()"
-          role="${ButtonRole.Tertiary}"
-        ></ht-button>
-        <div class="back-next">
-          <ht-button
-            class="back"
-            *ngIf="stepper.selectedIndex !== 0"
-            display="${ButtonStyle.Outlined}"
-            label="Back"
-            (click)="stepper.previous()"
-          ></ht-button>
-          <ht-button
-            class="next"
-            [label]="this.getActionButtonLabel | htMemoize: stepper.selectedIndex:steps"
-            (click)="this.nextOrSubmit(stepper)"
-            [disabled]="this.isNextDisabled(stepper.selectedIndex)"
-          ></ht-button>
-        </div>
-      </div>
+      <footer class="footer">
```

Comment:
this can be moved inside `ng-container` at L33 ?

Reply:
It has styles applied which is needed.